readthedocs/templates/core/version_list.html

```diff
@@ -1,6 +1,8 @@
+{% load projects_tags %}
+
 <div class="module-list-wrapper">
```

Comment:
I didn't find any use case for this file.
Can it be deleted?

Reply:
Hrm, I think so!